packages/fxa-content-server/app/scripts/views/settings/emails.js

```diff
@@ -20,7 +20,11 @@ const t = msg => msg;
 
 const EMAIL_INPUT_SELECTOR = 'input.new-email';
 const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
+const EMAIL_GROUP_SELECTOR = 'li.email-address';
 const EMAIL_REFRESH_DELAYMS = 350;
+// This is compared against all secondary email
+// records, both verified and unverified
+export const MAX_SECONDARY_EMAILS = 3;
```

Comment:
You can pull this into our constants file, https://github.com/mozilla/fxa/blob/34b98ab954a86e288676c083d3d6984df2781b27/packages/fxa-content-server/app/scripts/lib/constants.js#L157.